### PR TITLE
Feature/screen estate: fix overflowing and misaligned fields + avoid mis-clicking

### PR DESCRIPTION
**Pull request by mietcls**
Fixes https://github.com/ugent-library/biblio-backoffice/issues/744.

Commits seem huge; but the changes all have gone through the same restructuring.

### CODE REVIEW

**Comment by nics** [comment]
hi @mietcls this can't be right: `This branch is [2 commits ahead](https://github.com/ugent-library/biblio-backoffice/compare/dev...feature/screen-estate), [205 commits behind](https://github.com/ugent-library/biblio-backoffice/compare/feature/screen-estate...dev) dev.`
trying to merge a branch from main into dev?

**Comment by mietcls** [comment]
I think there's confusion about the flow we have. Can we clear this out? There are multiple visions on the branching and merging flow (e.g. main-branch-dev-main VS dev-branch-dev-main flow) and I thought I was adhering to the style proposed :-)

**Comment by mietcls** [comment]
@nics Fixed it by merging dev into this branch, is this okay?
I will revert to this flow until new consensus arrives:
- main-branch-main for hotfixes
- dev-branch-dev for features

File: views/dataset/confirm_republish.gohtml [hunk 1]
```diff
@@ -8,12 +8,17 @@
...
         </div>
         <div class="modal-footer">
-            <button class="btn btn-link modal-close">Cancel</button>
-            <button class="btn btn-success"
-                hx-post="{{pathFor "dataset_republish" "id" .Dataset.ID|querySet "redirect-url" .RedirectURL}}"
-                hx-headers='{"If-Match": "{{.Dataset.SnapshotID}}"}'
-                hx-swap="none"
-            >Republish</button>
+            <div class="bc-toolbar">
+                <div class="bc-toolbar-left">
```
Comment (nics) [inline views/dataset/confirm_republish.gohtml:12]: Tag isn't closed I think

File: views/dataset/confirm_delete_abstract.gohtml [hunk 2]
```diff
@@ -8,12 +8,17 @@
...
         </div>
         <div class="modal-footer">
-            <button class="btn btn-link modal-close">Cancel</button>
-            <button class="btn btn-danger"
-                hx-delete="{{pathFor "dataset_delete_abstract" "id" .Dataset.ID "abstract_id" .AbstractID}}"
-                hx-headers='{"If-Match": "{{.Dataset.SnapshotID}}"}'
-                hx-swap="none"
-            >Delete</button>
+            <div class="bc-toolbar">
+                <div class="bc-toolbar-left">
```
Comment (nics) [inline views/dataset/confirm_delete_abstract.gohtml:12]: Tag isn't closed

File: views/dataset/confirm_withdraw.gohtml [hunk 3]
```diff
@@ -8,12 +8,17 @@
...
         </div>
         <div class="modal-footer">
-            <button class="btn btn-link modal-close">Cancel</button>
-            <button class="btn btn-danger"
-                hx-post="{{pathFor "dataset_withdraw" "id" .Dataset.ID|querySet "redirect-url" .RedirectURL}}"
-                hx-headers='{"If-Match": "{{.Dataset.SnapshotID}}"}'
-                hx-swap="none"
-            >Withdraw</button>
+            <div class="bc-toolbar">
+                <div class="bc-toolbar-left">
```
Comment (nics) [inline views/dataset/confirm_withdraw.gohtml:12]: Tag isn't closed

File: views/dataset/edit_abstract.gohtml [hunk 4]
```diff
@@ -17,19 +17,26 @@
...
-                hx-headers='{"If-Match": "{{.Dataset.SnapshotID}}"}'
-                hx-include=".modal-body"
-                hx-swap="none"
-            >
-                Update abstract
-            </button>
-            {{end}}
+            <div class="bc-toolbar">
+                {{if .Conflict}}
+                <div class="bc-toolbar-left">
```
Comment (nics) [inline views/dataset/edit_abstract.gohtml:22]: Tag isn't closed

File: views/shared/_contributor_summary.gohtml [hunk 5]
```diff
@@ -5,12 +5,12 @@
     </div>
     <div class="bc-avatar-text">
       <h3>
-        {{.Contributor.Person.FullName}}
+        <span class="mr-3">{{.Contributor.Name}}</span>
```
Comment (nics) [inline views/shared/_contributor_summary.gohtml:8]: Should be
```
<span class="mr-3">{{.Contributor.Person.FullName}}</span>
```

**Review by nics — CHANGES_REQUESTED** [review]

**Review by nics — APPROVED** [review]

**Pull request merged**